src/main/java/App.java

```diff
@@ -6,29 +6,35 @@
 
 public class App {
     public static void main(String[] args) {
+
         BufferedReader in = null;
+
+
         try {
             URL url = new URL("https://favqs.com/api/qotd");
             HttpURLConnection con = (HttpURLConnection) url.openConnection();
 
             con.setRequestMethod("GET");
-
             in = new BufferedReader(new InputStreamReader(con.getInputStream()));
+
+
             String inputLine;
+            String output = "";
             while ((inputLine = in.readLine()) != null) {
-
-                System.out.println(inputLine);
+                Gson gson = new Gson();
+                Quote q = gson.fromJson(inputLine,Quote.class);
+               System.out.println(inputLine);
```

Comment:
This doesn't print out the quote, nicely formatted; it prints out the json, which isn't human-readable.